docs/tutorial/force_calibration.rst

```diff
@@ -396,6 +396,44 @@ However, if we do not provide the height above the surface, we can see that the
     Consequently, when this model is selected, this parameter affects both passive and active calibration.
     For more information on this see the :doc:`theory section on force calibration</theory/force_calibration/force_calibration>` section.
 
+.. _bead_bead_tutorial:
+
+Active calibration with two beads far away from the surface
+-----------------------------------------------------------
+
+.. warning::
+
+    The implementation of the coupling correction models is still alpha functionality.
+    While usable, this has not yet been tested in a large number of different scenarios.
+    The API can still be subject to change *without any prior deprecation notice*!
+    If you use this functionality keep a close eye on the changelog for any changes that may affect your analysis.
+
+When performing active calibration, we get a smaller fluid velocity around the beads than expected when calibrating with two beads in a dual trap configuration.
+This leads to a smaller voltage readout than expected if there's no coupling and therefore a higher displacement sensitivity (microns per volt).
+Failing to take this into account results in a bias.
+Pylake offers a function to calculate a correction factor to account for the lower velocity around the bead.
+Appropriate coupling correction factors for oscillation in x can be calculated as follows::
+
+    factor = lk.coupling_correction_2d(dx=5.0, dy=0, bead_diameter=bead_diameter, is_y_oscillation=False)
+
+Here `dx` and `dy` represent the horizontal and vertical distance between the beads, while the parameter `bead_diameter` refers to the bead diameter.
+Note that all three parameters have to be specified in the same spatial unit (meters or micron).
+The final parameter `is_y_oscillation` indicates whether the stage was oscillated in the y-direction.
+
+The obtained correction factor can be used to correct the calibration factors::
+
+    Rd_corrected = factor * calibration["Rd"].value
+    Rf_corrected = calibration["Rf"].value / factor
+    stiffness_corrected = calibration["kappa"].value / factor**2
+
+To correct a force trace, simply divide it by the correction factor::
+
+    corrected_force1x = f.force1x / factor
+
+.. note::
+
+    This coupling model is only valid far away from the surface.
```

Comment:
@JoepVanlier 50 um up or only in the middle?

Reply:
I don't expect there to be a sharp cutoff point. My guess it will probably depend on the ratio of bead radius to distance like Faxen's law does.

My guess is that there is some regime where you might get away with using Faxen's law and this model together, but I don't know how far you have to be for this to be a reasonable model.

Maybe we should rephrase it as, `This coupling model neglects effects from the surface.` to basically state what it does. Right now we don't have enough information to give more user guidance than that.

Reply:
I have rephrased it as: `This coupling model neglects effects from the surface. It is intended for measurements performed at the center of the flowcell.`

If I have some time, I will look at this a bit more with FEM.

Reply:
I compared it a bit with some FEM simulations near the surface. While it certainly isn't perfect, it seems that a superposition of Faxen and this model is still a lot better than doing nothing. That said, to stay on the safe side for now, I would only recommend it for bulk, and recommend people not to use it yet near the surface. I am not sure how many people would be doing two-bead assays close to the surface, so I'm not sure whether there is a use-case for it.

![image](https://github.com/lumicks/pylake/assets/19836026/017ae199-51fa-4883-bf64-2213e8cced6b)

Reply:
Sorry, I had somehow missed this comment. Yeah. For `Re << 1`, Navier Stokes reduces to just the Stokes equations and they are linear w.r.t. pressure and velocity. So yes, I would indeed expect this. But I feel the same way. We have not validated coupling behavior near the surface and I do not feel comfortable exposing that at this time.

Reply:
That's all right, I had forgotten about it myself 😆 
Goes on the TO DO list then :)